spec/controllers/account/admin/osbbs_controller_spec.rb

```diff
@@ -31,6 +32,7 @@
 
   describe 'GET#new' do
     it 'returns success and assigns osbb' do
+      sign_in simple
```

Comment:
so you just removed tests for admins with sing_in simple every where
how should we know admin functionality is working now?